subsys/bluetooth/controller/ll_sw/nordic/lll/lll_test.c

```diff
@@ -634,7 +639,12 @@ uint8_t ll_test_tx(uint8_t chan, uint8_t len, uint8_t type, uint8_t phy,
 	test_cte_len = cte_len;
 #endif /* CONFIG_BT_CTLR_DF_CTE_TX */
 
-	radio_switch_complete_and_disable();
+	/* Set the initial value of the TIFS. It will be tuned up after
+	 * the first packet transmission.
+	 */
+	radio_tmr_tifs_set(SCAN_INT_UNIT_US);
+	radio_switch_complete_and_b2b_tx(test_phy, test_phy_flags, test_phy, test_phy_flags);
+
```

Comment:
The second PDU will not be at a multiple of 625us slot, this is not correct.

Instead modify `calculate_tifs()` to calculate the tIFS using the PDU length and PHY used, use it here and in `isr_tx` so that first and consecutive PDUs will all be at 625us slot multiples.